bitnami/odoo/templates/tls-secrets.yaml

```diff
@@ -18,3 +20,25 @@ data:
 ---
 {{- end }}
 {{- end }}
+{{- if and .Values.ingress.tls (not .Values.ingress.certManager) }}
```

Comment:
We're changing this approach and introducing a parameter `ingress.selfSigned` for users interested on having self signed certificates to make it explicit. 
Check Jenkins, Kubeapps or RabbitMQ that are already using this approach.

Reply:
OK, applied.